go.sum

```diff
@@ -74,9 +76,8 @@ github.com/coreos/bbolt v1.3.2 h1:wZwiHHUieZCquLkDL0B8UhzreNWsPHooDAG3q34zk0s=
 github.com/coreos/bbolt v1.3.2/go.mod h1:iRUV2dpdMOn7Bo10OQBFzIJO9kkE559Wcmn+qkEiiKk=
 github.com/coreos/etcd v3.3.22+incompatible h1:AnRMUyVdVvh1k7lHe61YEd227+CLoNogQuAypztGSK4=
 github.com/coreos/etcd v3.3.22+incompatible/go.mod h1:uF7uidLiAD3TWHmW31ZFd/JWoc32PjwdhPthX9715RE=
+github.com/coreos/go-semver v0.2.0 h1:3Jm3tLmsgAYcjC+4Up7hJrFBPr+n7rAqYeSw/SZazuY=
```

Comment:
Why is this downgrading from v0.3.0?  I didn't see a related change in the satellite PR.

Reply:
It is not downgrading from `0.3.0` - `0.3.0` was simply removed as an indirect dependency in satellite and `go mod` added this bit - it's the dependency of `etcd` from the referenced [commit](https://github.com/etcd-io/etcd/blob/e694b7bb087538c146e188a29753967d189d202b/go.mod#L6).